Source/Scene/Globe.js

```diff
@@ -550,6 +569,36 @@ define([
         }
     };
 
+    /**
+     * @private
+     */
+    Globe.prototype.dirtyShaders = function() {
+        var defines = [];
+
+        var fragmentSources = [];
+        if (defined(this._material)) {
+            fragmentSources.push(this._material.shaderSource);
+            defines.push('APPLY_MATERIAL');
+
+            // Set the material uniform map to the materials
+            this._surface._tileProvider.uniformMap = this._material._uniforms;
+        } else {
+            this._surface._tileProvider.uniformMap = undefined;
+        }
+        fragmentSources.push(GlobeFS);
+
+        this._surfaceShaderSet.baseVertexShaderSource = new ShaderSource({
+            sources : [GroundAtmosphere, GlobeVS],
+            defines: defines
```

Comment:
Throughout, we would generally write `defines :` with a space before the semicolon.  The auto formatter in your IDE probably does this.  WebStorm does.